<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="todoEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>731</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>440</y>
      <width>741</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Todo</string>
    </property>
   </widget>
   <widget class="QListView" name="todoView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>721</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>701</width>
      <height>80</height>
     </rect>
    </property>
    <widget class="QPushButton" name="deleteButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="completeButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Complete</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
